src/System.Data.SqlClient/tests/FunctionalTests/project.json

```diff
@@ -4,6 +4,11 @@
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-24109-00",
     "System.Data.Common": "4.0.1-rc3-24109-00",
     "System.Text.RegularExpressions": "4.1.0-rc3-24109-00",
+    "System.Collections": "4.0.11-rc3-24109-00",
+    "System.Collections.Concurrent": "4.0.12-rc3-24109-00",
+    "System.Collections.NonGeneric": "4.0.1-rc3-24109-00",
```

Comment:
@saurabh500 That dependency is unnecessary, and I have removed it (and tested post-removal) in commit https://github.com/dotnet/corefx/pull/8420/commits/23de00d65e4e80cf7e28be84a797be906912c692